### PR TITLE
Add webStorage.html page

### PR DESCRIPTION
**Pull request by VerkhovtsovPavel**

### CODE REVIEW

File: src/main/resources/static/browserStorage.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Browser storages</title>
+</head>
+
+<body>
+  <p>
+    <button id="set-local-storage-item" onclick="window.localStorage.setItem('testKey', 'testLocalValue')">Set local storage item</button>
```
Comment (ikalinin1) [inline src/main/resources/static/browserStorage.html:11]: localKey

File: src/main/resources/static/browserStorage.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  </p>
+  <p>
+    <button id="set-session-storage-item" onclick="window.sessionStorage.setItem('testKey', 'testSessionValue')">Set session storage item</button>
```
Comment (ikalinin1) [inline src/main/resources/static/browserStorage.html:14]: sessionKey

File: src/main/resources/static/browserStorage.html (hunk 1 above)
Comment (ikalinin1) [reply src/main/resources/static/browserStorage.html:11]: I think it would help to troubleshoot the issue if there is any

**Review by ikalinin1 — APPROVED** [review]

**Review by uarlouski — APPROVED** [review]

File: src/main/resources/static/browserStorage.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Browser storages</title>
```
Comment (valfirst) [inline src/main/resources/static/browserStorage.html:6]: common name for this technology is `Web Storage`: https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API:

```suggestion
  <title>Web Storage</title>
```

Comment (valfirst) [reply src/main/resources/static/browserStorage.html:6]: also please update HTML file name in the same way

**Review by valfirst — CHANGES_REQUESTED** [review]

File: src/main/resources/static/browserStorage.html (hunk 3 above)
Comment (VerkhovtsovPavel) [reply src/main/resources/static/browserStorage.html]: Both the title and the file name have been updated.

**Review by valfirst — APPROVED** [review]

**Pull request merged**